configs/components/puppet-runtime.rb

```diff
@@ -40,6 +40,8 @@
     pkg.requires 'libyaml-cpp0_6'
   end
 
+  pkg.requires '/usr/bin/find' if platform =~ /^fedora/
```

Comment:
Hi. I don't think that this is Fedora specific. We just noticed it first on Fedora, but the dependency exists on every linux platform?